spoon-runner/src/main/java/com/squareup/spoon/SpoonDeviceRunner.java

```diff
@@ -138,7 +140,6 @@ private void printStream(InputStream stream, String tag) throws IOException {
 
   /** Execute instrumentation on the target device and return a result summary. */
   public DeviceResult run(AndroidDebugBridge adb) {
```

Comment:
The checkstyle error is form here - you'll ned to break this method apart into more sub-methods to reduce it's size.